src/treetensornetworks/ttn.jl

```diff
@@ -66,8 +66,8 @@ end
 # TODO: Implement `random_circuit_ttn` for non-trivial
 # bond dimensions and correlations.
 # TODO: Implement random_ttn for QN-Index
-function random_ttn(args...; kwargs...)
-  T = TTN(args...; kwargs...)
+function random_ttn(args...; link_space=1, kwargs...)
+  T = TTN(args...; link_space, kwargs...)
```

Comment:
Maybe instead `TTN` should use a default of `link_space=1`?

Reply:
I addressed this, but I am not entirely sure whether that could have unwanted side-effects?